panther_analysis_tool/command/migrate.py

```diff
@@ -138,10 +138,18 @@ def migrate(
     cache = analysis_cache.AnalysisCache()
 
     ancestor_commit: str | None = None
-    try:
-        ancestor_commit = git_helpers.get_forked_panther_analysis_common_ancestor()
-    except RuntimeError as err:
-        logging.debug("Failed to get forked panther analysis common ancestor: %s", err)
+
+    with Progress(
+        TextColumn("Checking when repo last updated:"),
+        BarColumn(),
+        transient=True,
+    ) as progress:
+        task = progress.add_task("checking_for_common_ancestor", total=None)
+        try:
+            ancestor_commit = git_helpers.get_forked_panther_analysis_common_ancestor()
```

Comment:
There was a weird lull is user feedback because this takes a bit of time so added a progress bar here.